processing file asmFiles/count_days.asm (1 of 24)
processing file asmFiles/fib.asm (2 of 24)
processing file asmFiles/mergesort_17.asm (3 of 24)
processing file asmFiles/mergesort17.asm (4 of 24)
processing file asmFiles/mergesort.asm (5 of 24)
--- memsim.hex	2023-09-25 18:06:04.434962004 -0400
+++ memcpu.hex	2023-09-25 18:06:04.417961346 -0400
@@ -80,60 +80,60 @@
 :04004F0010E00001BC
 :040050000800004A5A
 :0400510003E00008C0
-:0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
+:0400C000000000013B
+:0400C1000000000338
+:0400C2000000000436
+:0400C3000000000732
 :0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
-:0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
+:0400C5000000000730
+:0400C600000000072F
+:0400C700000000082D
+:0400C800000000082C
+:0400C900000000082B
+:0400CA000000000929
+:0400CB000000000A27
+:0400CC000000000B25
+:0400CD000000000C23
+:0400CE000000000F1F
+:0400CF00000000101D
+:0400D000000000111B
+:0400D1000000001516
+:0400D2000000001614
+:0400D3000000001613
+:0400D4000000001711
+:0400D500000000190E
+:0400D6000000001A0C
+:0400D7000000001B0A
+:0400D8000000001D07
+:0400D9000000002003
+:0400DA000000002002
+:0400DB000000002001
+:0400DC0000000021FF
+:0400DD0000000023FC
+:0400DE0000000024FA
+:0400DF0000000026F7
+:0400E00000000026F6
+:0400E10000000028F3
+:0400E20000000029F1
+:0400E3000000002FEA
+:0400E40000000033E5
+:0400E50000000036E1
+:0400E60000000038DE
+:0400E7000000003DD8
+:0400E8000000003DD7
+:0400E9000000003ED5
+:0400EA0000000040D2
+:0400EB0000000042CF
+:0400EC0000000047C9
+:0400ED000000004BC4
+:0400EE000000004DC1
+:0400EF0000000050BD
+:0400F00000000051BB
+:0400F10000000051BA
+:0400F20000000053B7
+:0400F3000000005AAF
+:0400F40000000063A5
+:0400F50000000064A3
 :0400F60000000030D6
 :0400F70000000032D3
 :0400F80000000041C3
@@ -145,69 +145,69 @@
 :0400FE000000005CA2
 :0400FF000000005CA1
 :040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
+:0401400000000003B8
+:0401410000000004B6
+:0401420000000007B2
 :0401430000000007B1
 :0401440000000007B0
 :0401450000000007AF
-:0401460000000007AE
+:0401460000000008AD
 :0401470000000008AC
 :0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
+:0401490000000009A9
+:04014A000000000AA7
+:04014B000000000BA5
+:04014C000000000CA3
+:04014D000000000F9F
+:04014E00000000109D
+:04014F00000000119B
+:040150000000001596
+:040151000000001694
 :040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
+:040153000000001791
+:04015400000000198E
+:040155000000001A8C
+:040156000000001B8A
+:040157000000001D87
+:040158000000002083
 :040159000000002082
 :04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
+:04015B00000000217F
+:04015C00000000237C
+:04015D00000000247A
+:04015E000000002677
 :04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
+:040160000000002873
+:040161000000002971
+:040162000000002F6A
+:040163000000003365
+:040164000000003661
+:04016500000000385E
+:040166000000003D58
+:040167000000003D57
+:040168000000003E55
+:040169000000004052
+:04016A00000000424F
+:04016B000000004749
+:04016C000000004B44
+:04016D000000004D41
+:04016E00000000503D
+:04016F00000000513B
+:04017000000000513A
+:040171000000005337
+:040172000000005A2F
+:040173000000006325
+:040174000000006423
+:040175000000003056
+:040176000000003253
+:040177000000004143
+:040178000000004241
+:04017900000000433F
+:04017A00000000443D
+:04017B000000005927
+:04017C000000005B24
+:04017D000000005C22
+:04017E000000005C21
 :04017F000000006418
 :043FFE0000000500BA
 :00000001FF
processing file asmFiles/mergesort_ordered.asm (6 of 24)
processing file asmFiles/mult.asm (7 of 24)
processing file asmFiles/mult_procedure.asm (8 of 24)
processing file asmFiles/problem.asm (9 of 24)
processing file asmFiles/search.asm (10 of 24)
processing file asmFiles/test.branch.asm (11 of 24)
processing file asmFiles/test.branchjump1.asm (12 of 24)
processing file asmFiles/test.halt.asm (13 of 24)
processing file asmFiles/test.hazard1.asm (14 of 24)
processing file asmFiles/test.hazard2.asm (15 of 24)
processing file asmFiles/test.hazard3.asm (16 of 24)
processing file asmFiles/test.hazard4.asm (17 of 24)
processing file asmFiles/test.hazard5.asm (18 of 24)
processing file asmFiles/test.hazard6.asm (19 of 24)
processing file asmFiles/test.jump.asm (20 of 24)
processing file asmFiles/test.ldst1.asm (21 of 24)
processing file asmFiles/test.ldst2.asm (22 of 24)
processing file asmFiles/test.loadstore.asm (23 of 24)
processing file asmFiles/test.rtype.asm (24 of 24)
